Makefile

```diff
@@ -802,17 +827,24 @@ app: appclean version data/osx/AppIcon.icns $(APPTARGET)
 	cp -R data/motd $(APPDATADIR)
 	cp -R data/credits $(APPDATADIR)
 	cp -R data/title $(APPDATADIR)
-	# bundle libc++ to fix bad buggy version on osx 10.7
-	LIBCPP=$$(otool -L $(APPTARGET) | grep libc++ | sed -n 's/\(.*\.dylib\).*/\1/p') && cp $$LIBCPP $(APPRESOURCESDIR)/ && cp $$(otool -L $$LIBCPP | grep libc++abi | sed -n 's/\(.*\.dylib\).*/\1/p') $(APPRESOURCESDIR)/
```

Comment:
I did intentionally comment this out, as it seemed for me too old (6 releases back from current Sierra). I don't want to argue, but would it not be good to gently push towards more modern OS versions and get rid of the hacks like this?
If you insist having it, I would need to update the `libs` directory. I, however have this library as of version 10.11.